libs/s25main/nodeObjs/noFighting.cpp

```diff
@@ -313,19 +296,13 @@ void noFighting::StartAttack()
         switch(gwg->GetGGS().getSelection(AddonId::ADJUST_MILITARY_STRENGTH))
         {
             case 0: // Maximale Stärke
-            {
                 results[i] = RANDOM.Rand(__FILE__, __LINE__, GetObjId(), soldiers[i]->GetRank() + 6);
-            }
-            break;
+                break;
```

Comment:
I don't like the different style here?
shouldn't be the `break` consistent - either on one line or on multiple lines?

Reply:
Only the default case is different. Clang-Format detects that it is a short line and puts it all in one line.

I guess we could move the comment to after the default: so clang-format has to use multiple lines. But well, not really keen on doing that now as it will retrigger a whole CI cycle and I don't think it's worth it. Next time we touch this...